### ISSUE TITLE
Replace NiArms RPKs with 3CB RPKs

### ISSUE THREAD

**Issue by JamesTheClarke** [CntoDev/compositions#154]
Due to the upcoming removal of @niarms_rpk the following loadouts have to be updated for the OPFOR Standard Faction:

- [x] AR Primary Weapon
- [x] AR Magazines (75rnd drum mags)
- [x] AAR Backpack mags (75rnd drum mags)
- [x] Fireteam Resupply Box
- [x] Platoon Resupply Crate

**Comment by JamesTheClarke** [comment]
Had to change AR backpack from Sidor to Knapsack to accommodate for larger mag size.

**Comment by JamesTheClarke** [comment]
AR carries 1 mag more than before because AAR could store 1 mag less.

**Comment by milivojm** [comment]
Hey guys, I'd like to raise a concern regarding mixing different mods with one example. 

RPK weapon inside 3CB is modeled as class `UK3CB_RPK`. It uses dozens of magazines defined in 3CB, Vanilla (Apex DLC, Contact DLC), RHS, HLC mods with many different parameters and characteristics which are hardly logical.

I'll paste here their config attributes for you to check and understand the underlying issue of mixing mods.

https://gist.github.com/milivojm/c018c871cc02bdef1995f46c98aad456

I really don't expect our mission makers to pay attention to these details and just shows that "same magazine in different mods" behaves differently. So my simple question is, why do we mix them?

**Comment by JamesTheClarke** [comment]
We've mixed them previously, just with NiArms. Now we're still mixing them with just with 3CB. Effectively nothing has changed and this has been present for over a year. The same is true for the MMG of TFN (BW mod) and some other assets we use (famously the PMC faction).

The mods are marked in Arsenal and Custom Factions have to be checked by MMM for these kind of errors so I don't see a major issue with mod mixes. Standard Factions are not allowed to be modified so there's no issue either.

TLDR: this has been a long-standing practice within MMT and if we really want to change that this would have a.) a big impact on MMT workflow and be would limit our variety quite a bit. In any case this should be discussed on the MMT Forums and not here, this is issue report just concerns the swap from mod a to mod b.

**Comment by milivojm** [comment]
Same 30 bullet magazine from 3CB weights twice as much as RHS one, 2.5 times as vanilla magazine for same weapon. Vanilla magazine will destroy the target (different damage model), etc. So if MM puts Syndikat as OPFOR, as player is best to remove mags from RPK and use vanilla ones rather found on a floor.

**Comment by JamesTheClarke** [comment]
I know and I see your point, but like I said: the same situation was present with NiArms RPK for a year and it didn't cause any major issues. The RPK is a good AR platform that Eastern factions are sorely lacking since we can't use the DLC Chinese variant.

As long as we don't run into real concrete issues (we fight 3CB factions way more often than Syndikat with their lighter drum mags) we shouldn't try to fix something that ain't broke. If we fight RHS factions the ARs can technically grab some 30rnd mags but in turn, they lose the sustained firepower of 75rnd drums - seems like a fair trade-off.

**Comment by freghar** [comment]
If you don't want to mix mods, use `rhs_weap_pm63` as the "RPK" (looks similar, front grip, overheats less, etc.) with `rhs_75Rnd_762x39mm` drums (or `rhs_75Rnd_762x39mm_tracer` tracers).

**Comment by freghar** [comment]
Just for info - the UK3CB version seems to be using 7.62x51mm :).

```
rhs_30Rnd_545x39_7N22_plum_AK:
ammo = "rhs_B_545x39_7N22_Ball";

rhs_B_545x39_7N22_Ball:
caliber = 0.939794;
hit = 9.8;
typicalSpeed = 915;
```
```
UK3CB_RPK74_60rnd_545x39:
ammo = "UK3CB_762x39_Ball_Red";

UK3CB_762x39_Ball_Red:
hit = 11.6;

UK3CB_762x39_Ball_Red inherits vanilla B_762x51_Ball:
hit = 11.6;
caliber = 1.6;
typicalSpeed = 800;
```
The sad part is that UK3CB has `UK3CB_545x39_Ball_Green` as `CfgAmmo` (and it matches RHS values), they just don't use it for the RPK mag.
